### PR TITLE
Adding notes for domain local groups + FQDN

### PR DESCRIPTION
**Pull request by joshts0**
Added note that domain local groups are not supported, nor is using FQDN in logins.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@joshts0 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/linux/sql-server-linux-active-directory-authentication.md [hunk 1]
```diff
@@ -259,6 +259,11 @@ CONTOSO.COM = {
 > [!NOTE]
 > While it is not recommended, it is possible to use utilities, such as **realmd**, that set up SSSD while joining the Linux host to the domain, while configuring **disablesssd** to true so that SQL Server uses openldap calls instead of SSSD for Active Directory related calls.
 
+> [!NOTE]
+> SQL Server login using FQDN (for example, CONTOSO.COM\Username) is not supported. Use CONTOSO\Username format.
```
[the remark this answers is not in the document]
Comment (VanMSFT) [reply docs/linux/sql-server-linux-active-directory-authentication.md:263]: Hey Amit, for this sentence, I think Josh's usage is correct. I would like to go with his format as we do use CONTOSO in various formats in our docs.

File: docs/linux/sql-server-linux-active-directory-authentication.md [hunk 2]
```diff
@@ -259,6 +259,11 @@ CONTOSO.COM = {
...
+> 
+> Due to a KRB5 limitation, you will not be able to add SQL Server logins from Domain Local Groups.
```
[the remark this answers is not in the document]
Comment (VanMSFT) [reply docs/linux/sql-server-linux-active-directory-authentication.md:265]: How about this?

```suggestion
> SQL Server logins from Domain Local Groups is not supported. Use Global Security Domain Groups instead.
```

**Review by VanMSFT — COMMENTED** [review]
@amvin87 - please see my response. Thanks!

**Comment by amvin87** [comment]
Sure, I am good with the changes. Lets merge them please.

**Comment by PRMerger13** [comment]
@VanMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/linux/sql-server-linux-active-directory-authentication.md [hunk 3]
```diff
@@ -259,6 +259,11 @@ CONTOSO.COM = {
 > [!NOTE]
 > While it is not recommended, it is possible to use utilities, such as **realmd**, that set up SSSD while joining the Linux host to the domain, while configuring **disablesssd** to true so that SQL Server uses openldap calls instead of SSSD for Active Directory related calls.
 
+> [!NOTE]
+> SQL Server login using FQDN (for example, CONTOSO.COM\Username) is not supported. Use CONTOSO\Username format.
+> 
+> SQL Server logins from Domain Local Groups is not supported. Use Global Security Domain Groups instead.
```
Comment (VanMSFT) [inline docs/linux/sql-server-linux-active-directory-authentication.md:265]:
```suggestion
> SQL Server logins from Domain Local Groups are not supported. Use Global Security Domain Groups instead.
```

**Comment by PRMerger13** [comment]
@VanMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by VanMSFT** [comment]
#sign-off

**Comment by PRMerger16** [comment]
@VanMSFT: I'm sorry - only the author of this article, @Dylan-MSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by Dylan-MSFT** [comment]
#sign-off

**Comment by PRMerger19** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**